src/App.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function isInclude(word: string, part: string) {
```

Comment:
Get used to name your functions according to these rules, so it's clear that it is a function, not some boolean value
![image](https://user-images.githubusercontent.com/96409563/219464631-bf185223-875b-4ab8-b45d-b71ac970cc67.png)